tests/Drupal/Commenting/TodoCommentUnitTest.inc

```diff
@@ -7,9 +7,9 @@
  * These are valid examples.
  *
  * @todo Valid.
- * @todo valid with lower-case first letter
- * @todo $can start with a $
```

Comment:
this should be valid, one could document a variable name like "@todo $entity could be NULL here, fix this later in foo_fcuntion()."